lib/presentation/widgets/logo_form.dart

```diff
@@ -51,6 +51,7 @@ class _MyLogFormWidgetState extends State<LogoForm> {
               if (formKey.currentState!.validate()) {}
             },
             text: LocaleKeys.sing_in.tr(),
+            isEnabled: true,
```

Comment:
it has a default value of true. no need to pass it here

Reply:
I will change it when I start to work  with the login